services/iam/doc/openapi.json

```diff
@@ -1044,6 +1076,34 @@
           "password"
         ]
       },
+      "Register": {
+        "title": "Register data",
+        "type": "object",
+        "properties": {
+          "username": {
+            "type": "string"
+          },
+          "firstname": {
+            "type": "string"
+          },
+          "lastname": {
+            "type": "string"
+          },
+          "companyName": {
```

Comment:
companyName as the only parameter in camelCase is cumbersome. change in all locations to lowercase like the other parameters